src/foundry/common/packages/base-package.d.mts

```diff
@@ -75,6 +75,10 @@ declare namespace BasePackage {
 
     banner: fields.StringField<optionalString>;
 
+    packageName: fields.StringField<optionalString>;
+
+    packageType: fields.StringField<optionalString>;
```

Comment:
This is incorrect. These properties are set when PackageCompendiumPacks is initialized, rather than being part of the schema.

Reply:
Thanks! I moved these to `game.data.packs` and updated their type definitions, hopefully that's closer to correct.